### ISSUE TITLE
Use the siren HTTP API of comit-rs for accept/decline actions

### ISSUE THREAD

**Issue by D4nte** [comit-network/comit-i#45]
Take inspiration from the comit-rs e2e test suite and implement accept/decline actions.
The "source" in the comit-rs testsuite is the "user" in comit-i.
Use the `class` attribute of the `fields` to select the correct component, e.g. `ethereum-address`.

Note use http API mocks as we do in Bobtimus

DoD:
- All HTTP requests generated for actions are generated from Siren actions
- No special-case handling of any action (accept/decline/deploy/fund/refund/redeem) should all be handled the same way

**Comment by da-kami** [comment]
This issue includes the necessary changes so that comit-i can fetch data from comit-rs again.

**Comment by thomaseizinger** [comment]
Implementation concept:

- Use routing instead of DialogState

Dynamic route for action like here: https://reacttraining.com/react-router/web/example/recursive-paths

```jsx
<Route path={`${swapLink}/:action`} render={() => (
	<ActionDialog action={sirenAction}>
)} />
```

This route is inserted at the end of the each `<SwapRow />`

ActionDialog will display action:

- if action has fields: Display form for field, button text should be name from Action
- if action has no fields -> execute right away + inspect response. if response is ledger-action, display it

Open questions:

- does this work with deep-linking to the action?

**Comment by bonomat** [comment]
Resolved in #74.
